driver.sh

```diff
@@ -170,10 +165,17 @@ check_dependencies() {
   oldest_llvm_version=7
   latest_llvm_version=$(curl -LSs https://raw.githubusercontent.com/llvm/llvm-project/master/llvm/CMakeLists.txt | grep -s -F "set(LLVM_VERSION_MAJOR" | cut -d ' ' -f 4 | sed 's/)//')
 
-  for readelf in $(gen_bin_list llvm-readelf) llvm-readelf; do
-    command -v ${readelf} &>/dev/null && break
+  for llvm_version in $(seq "${latest_llvm_version}" -1 "${oldest_llvm_version}"); do
+    debian_llvm_bin=/usr/lib/llvm-${llvm_version}/bin
+    if [[ -d ${debian_llvm_bin} ]]; then
+      export PATH=${debian_llvm_bin}:${PATH}
+      break
```

Comment:
else die

Reply:
If I understand correctly, you mean something like this?

```diff
diff --git a/driver.sh b/driver.sh
index 3e73efa..d617bfe 100755
--- a/driver.sh
+++ b/driver.sh
@@ -170,6 +170,9 @@ check_dependencies() {
     if [[ -d ${debian_llvm_bin} ]]; then
       export PATH=${debian_llvm_bin}:${PATH}
       break
+    else
+      echo "Can't find ${debian_llvm_bin}"
+      exit 1
     fi
   done

```

If so, that won't work.

1. We're iterating in a loop.

2. The user might not be on Debian/Ubuntu.

We'll catch these tools not being available down below though.